classes/components/listPanels/CatalogListPanel.inc.php

```diff
@@ -100,7 +101,7 @@ public function getConfig()
             while (!$seriesResult->eof()) {
                 $seriesObj = $seriesResult->next();
                 [$seriesSortBy, $seriesSortDir] = explode('-', $seriesObj->getSortOption());
-                $seriesSortDir = empty($seriesSortDir) ? $catalogSortDir : $seriesSortDir == SORT_DIRECTION_ASC ? 'ASC' : 'DESC';
+                $seriesSortDir = (empty($seriesSortDir) ? $catalogSortDir : $seriesSortDir == SORT_DIRECTION_ASC) ? 'ASC' : 'DESC';
```

Comment:
I'm not sure if this is the right place for the parentheses. It's a convoluted syntax, but I think what's intended here is:

```php
$a = empty($seriesSortDir) ? $catalogSortDir : ($seriesSortDir == SORT_DIRECTION_ASC ? 'ASC' : 'DESC');
```

Same with line 82 above...